conferences/models.py

```diff
@@ -91,6 +91,15 @@ def end_date(self):
     def __str__(self):
         return 'Zosia {}'.format(self.start_date.year)
 
+    @property
+    def is_rooming_open(self):
+        return self.rooming_start <= datetime.now().date() <= self.rooming_end
+
+    def can_room(self, user):
+        time_with_bonus = datetime.now() + timedelta(0, 3600*user.bonus_minutes)
+        return user.payment_accepted and \
+            time_with_bonus >= datetime.combine(self.rooming_start, datetime.min.time())
```

Comment:
I hope you know what you're doing in that function :) imo you should add docstrings to that function and maybe some helptext to `self.rooming_start` definition